.cspell.json

```diff
@@ -1,3 +1,3 @@
 {
-  "ignorePaths": ["**/words.txt"]
+  "ignorePaths": ["**/words.txt", ".github/CODEOWNERS"]
```

Comment:
Should we ignore `.github/CODEOWNERS` in https://github.com/tier4/autoware-spell-check-dict/blob/main/.cspell.json rather than here?

Reply:
Or if you use email instead, I believe you don't need to ignore the file.
https://github.com/autowarefoundation/autoware.universe/blob/main/.github/CODEOWNERS

Reply:
Thank you for your feedback. I replaced account name by account e-mail address.